frontend/src/app/progress/progress.component.html

```diff
@@ -1,7 +1,16 @@
 <div class="progress-wrapper" role="alert" aria-live="assertive">
   <div class="progress-inner">
-    <p>Submitting your application.</p>
-    <span class="ds-c-spinner ds-u-fill--background-inverse ds-u-color--base-inverse ds-c-spinner--filled ds-u-valign--middle" aria-valuetext="Loading" role="progressbar"></span>
-    <p>Uploading file {{ fileUploadProgress }} of {{ numberOfFiles }}.</p>
+    <div *ngIf="!fileUploadError">
```

Comment:
It would be nice to have in the readme a bit about this component - where it came from.